.python-version

```diff
@@ -0,0 +1 @@
+3.7.12
```

Comment:
Why is this necessary? We don't use 3.7 for other repos that use dependabot, do we?

Reply:
We do!  The docs repo is 3.7 only because that's the latest CloudFlare Pages supports.  Dependabot appears to run with either 3.9 or 3.10 and this is the only way to tell it otherwise.  Since this plugin needs to run in the docs checkout on CloudFlare it's also tied to 3.7.

Reply:
I think I understand now. In which case I think the commit message is misleading. I read it as "pin to 3.7 because dependabot needs that version", but I think it's more like "tell dependabot what Python version to use".

Reply:
Good point, message updated 👍